source/administration/changelog.md

```diff
@@ -4,6 +4,146 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.28 - [Feature Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
+
+**Release day: 2020-10-16**
+
+### Compatibility
+ - PostgreSQL ended long-term support for [version 9.4 in February 2020](https://www.postgresql.org/support/versioning). Mattermost is officially supporting PostgreSQL version 10 with v5.26 release as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. We plan on fully deprecating PostgreSQL 9.4 In our v5.30 release (December 16). Please follow the instructions under the Upgrading Section within [the PostgreSQL documentation](https://www.postgresql.org/support/versioning/).
+ - Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.19 has come to the end of its lifecycle. Upgrading to server v5.25 or later is required.
+
```

Comment:
Please add

 - TLS versions 1.0 and 1.1 have been deprecated by browser vendors. Starting in v5.31 (January 16th) Mattermost servers will return an error for these TLS versions and System admins will need to explicitly add these versions in their configuration to continue to use them (see [ticket](https://mattermost.atlassian.net/browse/MM-28592)).  We recommend upgrading to TLS version 1.2 or higher.